lib/spack/spack/util/module_cmd.py

```diff
@@ -195,9 +195,11 @@ def match_pattern_and_strip(line, pattern, strip=[]):
     def match_flag_and_strip(line, flag, strip=[]):
         flag_idx = line.find(flag)
         if flag_idx >= 0:
-            end = line.find(' ', flag_idx)
-            if end >= 0:
-                path = line[flag_idx + len(flag):end]
+            separators = (' ', '"', "'")
+            occurrences = [line.find(s, flag_idx) for s in separators]
```

Comment:
If this is supported by all Python versions going back to Python 2.6:
```suggestion
            occurrences = [line.find(s, start=flag_idx) for s in separators]
```
I think it could improve readability. Can you please double check?

Reply:
vermin doesn't complain, let's see about ci

Reply:
```E   TypeError: find() takes no keyword arguments``` doesn't seem to work :(